Create ui-c8y-1023-23-0-make-the-message-banner-form-branding-dissmissable-graft-release-cd-10997.md (#4176)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1023-23-0-make-the-message-banner-form-branding-dissmissable-graft-release-cd-10997.md

---
date: ""

> title: Make the message banner form branding dismissable

Reviewer: ```suggestion
title: Message banner can now be dismissed temporarily or permanently
```

> title: Message banner can now be dismissed temporarily or permanently

Reviewer 2: ```suggestion
title: Custom message banners in Branding can now be acknowledged and dismissed
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65885
version: 1023.23.0
---

> The message banner displayed in your application can now be dismissed by users in two different ways. Previously, the message banner reappeared after every application refresh, which could be disruptive to the user experience. Now you can dismiss it temporarily using the **Close** button, which will hide the banner until the next refresh. To permanently dismiss the banner, click **Acknowledge and close**. This stores the dismissal in the browser's local storage, so the banner does not reappear unless the browser cache is cleared, the user switches to a different browser, or a new message banner is configured in your branding settings. This change gives users more control over their experience while allowing administrators to ensure important messages are still communicated effectively through the branding configuration.

Reviewer: ```suggestion
The message banner displayed in the application can now be dismissed by users in two different ways. Previously, the message banner reappeared after every application refresh, which could be disruptive to the user experience. Now you can dismiss it temporarily using the **Close** button, which will hide the banner until the next refresh. To permanently dismiss the banner, click **Acknowledge and close**. This stores the dismissal in the browser's local storage, so the banner does not reappear unless the browser cache is cleared, the user switches to a different browser, or a new message banner is configured in your branding settings. This change gives users more control over their experience while allowing administrators to ensure important messages are still communicated effectively through the branding configuration.
```

> The message banner displayed in the application can now be dismissed by users in two different ways. Previously, the message banner reappeared after every application refresh, which could be disruptive to the user experience. Now you can dismiss it temporarily using the **Close** button, which will hide the banner until the next refresh. To permanently dismiss the banner, click **Acknowledge and close**. This stores the dismissal in the browser's local storage, so the banner does not reappear unless the browser cache is cleared, the user switches to a different browser, or a new message banner is configured in your branding settings. This change gives users more control over their experience while allowing administrators to ensure important messages are still communicated effectively through the branding configuration.

Reviewer 2: ```suggestion
Users can now permanently dismiss custom messages configured under Branding > Message Banner. Previously, these messages would reappear every time the page was refreshed, which could disrupt the user experience.

We have introduced two ways for users to clear these messages from the top of their screen:
- Close: Temporarily hides the message. It will reappear the next time the page is refreshed.
- Acknowledge and close: Confirms the user has read the message. The application will remember this preference for the specific browser being used and will not show the message again.

Note: The message will reappear if the administrator updates the banner configuration with a new text or if the user clears their browser cache.
```

> The message banner displayed in your application can now be dismissed by users in two different ways. Previously, the message banner would reappear after every application refresh, which could be disruptive to the user experience. Now you have the option to dismiss it temporarily using the 'Close' button, which will hide the banner until the next refresh, or to permanently dismiss it using the 'Acknowledge and close' button, which stores the dismissal in the browser's local storage so the banner will not reappear unless the browser cache is cleared, the user switches to a different browser, or a new message banner is configured in your branding settings. This gives users more control over their experience while allowing administrators to ensure important messages are still communicated effectively through the branding configuration.

Reviewer: ```suggestion
The message banner displayed in your application can now be dismissed by users in two different ways. Previously, the message banner reappeared after every application refresh, which could be disruptive to the user experience. Now you can dismiss it temporarily using the **Close** button, which will hide the banner until the next refresh. To permanently dismiss the banner, click **Acknowledge and close**. This stores the dismissal in the browser's local storage, so the banner does not reappear unless the browser cache is cleared, the user switches to a different browser, or a new message banner is configured in your branding settings. This change gives users more control over their experience while allowing administrators to ensure important messages are still communicated effectively through the branding configuration.
```

---
date: ""

> title: Custom message banners in Branding can now be acknowledged and dismissed

Reviewer: ```suggestion
title: Custom message banners in branding can now be acknowledged and dismissed
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65885
version: 1023.23.0
---

> Users can now permanently dismiss custom messages configured under Branding > Message Banner. Previously, these messages would reappear every time the page was refreshed, which could disrupt the user experience.

Reviewer: ```suggestion
Users can now permanently dismiss custom messages configured under **Branding** > **Message Banner**. Previously, these messages reappeared every time the page was refreshed, which could disrupt the user experience.
```

> We have introduced two ways for users to clear these messages from the top of their screen:

Reviewer: ```suggestion
Users can clear these messages from the top of their screen in two ways:
```
